### DISCUSSION TITLE
Can I load locales from different paths?

### DISCUSSION THREAD

**Discussion by ivansevillaa** [vinissimus/next-translate#875] [Q&A]
The default is to load the locale from `locales/en/user.json`

I want to load from wherever I want for example from:
`pages/user/i18n.json`

But is probably I have nested routes, so if inside user I have another screen I'd like to load the locale from the same way:
`pages/user/settings/i18n.json`

Is possible?

**Comment by aralroca** [comment]
You can implement [`loadLocaleFrom`](https://github.com/vinissimus/next-translate#create-your-namespaces-files) loading the locales as you want.  However, there is an issue using dynamic import with Next.js 12 compiler that we are trying to fix (there are currently a [workaround](https://github.com/vinissimus/next-translate/issues/851#issuecomment-1173611946))

**Reply by ivansevillaa** [reply]
Yes, but. Can I have a dynamic way to do that? For example for the first example `pages/user/en/i18n.json` I can do: 
```
"loadLocaleFrom": (lang, ns) =>
    import(`/pages/${ns}/${lang}/i18n.json`).then((m) => m.default)
```
But that doesn't work for `pages/user/settings/en/i18n.json`

**Reply by aralroca** [reply]
It should work if you use `user/settings` as namespace.

```js
const { t, lang } = useTranslation('user/settings')
```
